expyfun/_experiment_controller.py

```diff
@@ -948,7 +955,7 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
         ExperimentController.wait_for_presses
         """
         return self._response_handler.get_presses(live_keys, timestamp,
-                                                  relative_to)
+                                                  relative_to, type)
```

Comment:
@drammock, are you OK with `kind` instead of `type`?
On Apr 5, 2016 05:35, "Eric Larson" <notifications@github.com> wrote:

> In expyfun/_experiment_controller.py
> <https://github.com/LABSN/expyfun/pull/276#discussion_r58529575>:
>
> > @@ -948,7 +955,7 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
> >          ExperimentController.wait_for_presses
> >          """
> >          return self._response_handler.get_presses(live_keys, timestamp,
> > -                                                  relative_to)
> > +                                                  relative_to, type)
>
> Don't shadow builtins (don't use name type, perhaps kind instead)
>
> Also, I don't see type value- or type-checked anywhere, so currently a
> user passing a typo type='pressees will instead get some ambiguous
> KeyError instead of a clear message about what values are accepted
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LABSN/expyfun/pull/276/files/5b44b26c297ec88d338c98a4f771447be71ef91e#r58529575>
>